entries/u/up.com.au.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Up": {
+    "domain": "up.com.au",
+    "tfa": [
+      "sms"
```

Comment:
Can you provide a screenshot of the SMS 2FA setup and/or login process? I can't find any mention of it on their website.

Please make sure to blur or crop out any personal information.

![Screenshot upload guide](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)

Reply:
![image](https://user-images.githubusercontent.com/7688583/210167717-396ec5b8-ba9a-4e18-b745-8eb9e93a53e9.jpeg)